src/vs/workbench/electron-browser/main.contribution.ts

```diff
@@ -200,6 +200,11 @@ let properties: { [path: string]: IJSONSchema; } = {
 		'type': 'boolean',
 		'default': false,
 		'description': nls.localize('showFullPath', "If enabled, will show the full path of opened files in the window title.")
+	},
```

Comment:
Two things:
* this setting should be wrapped around a isLinux/isWindows check so that it does not show up on macOS where it is not supported
* I find the setting name a bit confusing because `autoHideMenuBar` is not really clear other than that it matches the Electron method we are using it for. I would try to change it to the intent it actually does (if this option is set to true, you can press Alt key to show the menu even if the menu is hidden)